sandboxes/analysis/output_error_monitor.lua

```diff
@@ -0,0 +1,57 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Monitors Parquet output plugin for errors, alerting when they pass a given threshold.
+
+```lua
+filename = 'parquet_output_error_monitor.lua'
+message_matcher = 'Type == "hindsight.plugins"'
```

Comment:
This plugin is not designed to handle stats from multiple plugins at once.  The matcher would also need a `Fields[name] == 'input.example'`